modules/secure-logging/Makefile.am

```diff
@@ -0,0 +1,30 @@
+module_LTLIBRARIES					+=	\
+	modules/secure-logging/libsecure-logging.la
+
+EXTRA_DIST += modules/secure-logging/CMakeLists.txt
+
+modules_secure_logging_libsecure_logging_la_CFLAGS	=	\
+	-I$(top_srcdir)/modules/secure-logging	                \
+	-I$(top_builddir)/modules/secure-logging
+
+modules_secure_logging_libsecure_logging_la_SOURCES		=	\
+	modules/secure-logging/secure-logging.c                         \
+	modules/secure-logging/slog.c
```

Comment:
You have to include the header files as  well here, so the automake/autotools could track those files and include into the source tarball (this is the reason why the first job fails).